MAUI/DataGrid/selection.md

```diff
@@ -556,4 +556,108 @@ By default, the keyboard navigation will be enabled when setting the selection a
                     ItemsSource="{Binding OrderInfoCollection}" >
 </syncfusion:SfDataGrid>
 {% endhighlight %}
+{% endtabs %}
+
+
+## Customizing the selection controller behavior
+
+For customizing the selection controller behavior, initially create the instance for selection controller then, we have to create a class by inheriting the `DataGridRowSelectionController`. And then override the the methods you want to customize.
+
+### Changing Enter key behavior
+
+By default, pressing the <kbd>Enter</kbd> key moves the current cell focus to the next cell in the same column. To change this behavior, you can override the relevant selection controllers based on the NavigationModes.
```

Comment:
By default, while pressing `Enter` key, the current cell will be moved to the next focused cell in the same column. You can change the behavior by overriding the `ProcessKeyDown` method in the custom selection controller.

Reply:
Updated